nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -229,7 +252,9 @@ protected MongoDatabase getDatabase(final ProcessContext context, final FlowFile
     }
 
     protected MongoCollection<Document> getCollection(final ProcessContext context, final FlowFile flowFile) {
-        final String collectionName = context.getProperty(COLLECTION_NAME).evaluateAttributeExpressions(flowFile).getValue();
+        final String collectionName = flowFile == null
```

Comment:
I think this should be != like the getDatabase() call above?

Reply:
Done.